Pipfile

```diff
@@ -10,6 +10,10 @@ pytest-datadir = "*"
 
 [packages]
 miniwdl = "*"
+urllib3 = ">=1.26.18"
+certifi = ">=2022.12.07"
+requests = ">=2.31.0"
+setuptools = ">=65.5.1"
```

Comment:
There should be an accompanying update to the Pipfile.lock.

Rather than adding and updating the dependencies here, can you try running "pipenv update". This should update the Pipfile.lock with the latest dependency versions.

Reply:
Done